src/components/group-management/leave-group-dialog/index.test.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { LeaveGroupDialog, Properties } from '.';
+
+describe('InviteDialog', () => {
+  const subject = (props: Partial<Properties>) => {
+    const allProps: Properties = {
+      name: '',
+      status: 0,
+      onClose: () => {},
+      onLeave: () => {},
+      ...props,
+    };
+
+    return shallow(<LeaveGroupDialog {...allProps} />);
+  };
+
+  it('renders the name of group', function () {
```

Comment:
What if the group isn't named?

Reply:
oh I didn't consider that. For now, I am just displaying "Are you sure you want to leave this group?" (instead of the name). Will also ask Ethan and will update in followup if any changes required.